bin/check-outline

```diff
@@ -1,54 +1,29 @@
 #!/usr/bin/env node
 
-const {docPagesDirectory} = require('../constants');
-const {lstatSync, readdirSync, readFileSync, writeFile, statSync} = require('fs');
-const {join, resolve} = require('path');
+const {docPagesDirectory, versions} = require('../constants');
+const {readdirSync, readFileSync, statSync} = require('fs');
+const {join} = require('path');
 const jsYaml = require('js-yaml');
 
 const loadedOutline = jsYaml.safeLoad(readFileSync(`${__dirname}/../config/outline.yaml`, 'utf8'));
-const directories = (() => readdirSync(docPagesDirectory).filter(
-        folder => statSync(join(docPagesDirectory, folder)).isDirectory() && '.' !== folder[0])
-)();
 
-const outlineDirectories = Object
-    .keys(loadedOutline.chapters)
-    .map(key => loadedOutline.chapters[key].path);
-
-const oddDirectories = {
-    missingDirectories: directories.filter(folder => !outlineDirectories.includes(folder)),
-    extraDirectories: outlineDirectories.filter(folder => !directories.includes(folder)),
-};
-
-if (oddDirectories.missingDirectories.length > 0) {
-    console.warn('\x1b[31m', `The following "path" entries are missing in "outline.yaml", please add them with the relevants "title" and "items" keys: ${oddDirectories.missingDirectories.join(', ')}`, '\x1b[37m');
-}
-
-const newOutline = loadedOutline.chapters.filter(folder => !oddDirectories.extraDirectories.includes(folder.path));
-
-newOutline
```

Comment:
Is this removal wanted?

Reply:
yes.

Reply:
Fact is, at first, it only was used to re-write compete the initial doc.
But since we are now parsing multiple version at the same time it's not really relevant.
The way it works now is more like, here is what I know that should be displayed, But if there is any thing unknown add it anyway afterward, so in any case we respect the order and we don't forgot any new pages that would be forgotten if not referenced here.

Reply:
The doc is now a bit more loosely coupled with the config.

Reply:
I'm not fully sure to understand but I trust you :slightly_smiling_face: